e2e_tests/README.md

```diff
@@ -0,0 +1,48 @@
+# Tests
+
+The project includes automated tests for testing the ASG Sync tool in an AWS environment (Azure support will be added later). The tests are written in Python3, use the pytest framework to run the tests and utilize boto3 to call the AWS API.
+
+Note: for now this is for internal use only, as AWS stack configuration is done outside of testing framework.
+
+Below you will find the instructions on how to run the tests against a cloud provider.
+
+## Running Tests
+
+### Prerequisites:
+
+* AWS stack prepared. 
+* AWS access key and AWS secret access key. 
+* Python3 or Docker.
+
+#### Step 1 - Set up the environment
+
+* Either create|update ~/.aws/credentials file or set the AWS_SHARED_CREDENTIALS_FILE environment variable pointing to your own location. This file is an INI formatted file with section names corresponding to profiles. Tests use 'default' profile. The file [credentials](data/credentials) is a minimal example of such a file.
+
+#### Step 2 - Run the Tests
+ 
+Run the tests:
+* Use local Python3 installation:
+    ```bash
+    $ cd e2e_tests
+    $ pip3 install -r requirements.txt
+    $ python3 -m pytest --nginx-api=nginx_plus_api_url
+    ```
+* Use Docker:
+    ```bash
+    $ cd e2e_tests
+    $ make build
+    $ make run-tests AWS_CREDENTIALS=abs_path_to_creds_file NGINX_API=nginx_plus_api_url
```

Comment:
I try to run it by got an error:
```
make run-tests AWS_CREDENTIALS=<REDACTED>/e2e_tests/data/credentials  NGINX_API=http://demo.nginx.com/api
. . .
docker run --rm -v <REDACTED>/e2e_tests/data/:~/.aws/credentials test-runner:edge --nginx-api=http://demo.nginx.com/api --aws-region=us-east-2
docker: Error response from daemon: invalid volume specification: '<REDACTED>/e2e_tests/data/:~/.aws/credentials': invalid mount config for type "bind": invalid mount path: '~/.aws/credentials' mount path must be absolute.
```

Reply:
What is `<REDACTED>` in your example (just in case, there should be an absolute path). But I agree there is a problem with running in the docker, I'll fix it.